Neos.Media/Classes/Domain/Model/Adjustment/ResizeImageAdjustment.php

```diff
@@ -456,6 +456,9 @@ protected function resize(ImagineImageInterface $image, string $mode = ImageInte
             throw new \InvalidArgumentException('Invalid mode specified', 1574686891);
         }
 
+        $autorotateFilter = new \Imagine\Filter\Basic\Autorotate();
```

Comment:
I am currently trying to untangle the complete image processing workflow ... Wouldn't it make sense to put it into a separate rotate adjustment? Is there no scenario where you don't want to have it rotated or where it is breaking? Should the adjustment be configurable?

Reply:
@daniellienert The autorotate filter does nothing if no exif orientation is set an in other cases it applies the transformation to the image so following filters do not have to deal with it. (https://github.com/avalanche123/Imagine/blob/develop/src/Filter/Basic/Autorotate.php) as far as i see this filter is meant to be applied to all images.

The only way that could cause problems i can imagine is when some code would transfer the original exif tags to the generated image-variant. Not sure wether such things exist and i would assume that the result would be broken in that case today aswell because the calculations are performed on flipped dimensions.

But i am far from sure that this is the correct way to apply this filter. The only thing i know is that this filter has to be applied before calculating dimensions and crops to support exif-orientations.

Reply:
Found this in the notes of the Autorotate filter `Your attention please: This filter requires the use of the ExifMetadataReader to work.` and in ExifMetadataReader it states `Using the exif metadata reader adds a significant overhead to image processing.`

Still think that it is still more important to properly read images and doubt that the exif parsing can be that significant.

Reply:
Yap. We have quite some projects where editors complain about this bug